### PR TITLE
feat(ios): iOS support/split appium out

### PR DESCRIPTION
**Pull request by sjelin**
Closes https://github.com/angular/webdriver-manager/issues/47 and https://github.com/angular/webdriver-manager/issues/48

### CODE REVIEW

File: lib/cmds/start.ts [hunk 1]
```diff
@@ -204,9 +211,9 @@ function killAndroid() {
 // Manage appium process
 let appiumProcess: childProcess.ChildProcess;
 
-function startAppium(outputDir: string, port: string) {
+function startAppium(outputDir: string, binary: Binary, port: string) {
   logger.info('Starting appium server');
-  appiumProcess = childProcess.spawn(path.join(outputDir, 'appium',
+  appiumProcess = childProcess.spawn(path.join(outputDir, binary.filename('', ''),
```
Comment (cnishina) [inline lib/cmds/start.ts]: I believe the two parameters are optional and you can use `binary.filename()`.

File: lib/cmds/update.ts [hunk 2]
```diff
@@ -25,6 +25,7 @@ let prog = new Program()
     .addOption(Opts[Opt.ALTERNATE_CDN])
     .addOption(Opts[Opt.STANDALONE])
     .addOption(Opts[Opt.CHROME])
+    .addOption(Opts[Opt.IOS])
```
Comment (cnishina) [inline lib/cmds/update.ts]: Check to see if the user is on os.type() === 'Darwin' before adding the iOS options.

File: lib/cmds/start.ts (hunk 1 above)
Comment (sjelin) [inline lib/cmds/start.ts]: You're right! fixing

File: lib/cmds/start.ts [hunk 3]
```diff
@@ -23,8 +24,10 @@ let prog = new Program()
     .addOption(Opts[Opt.VERSIONS_STANDALONE])
     .addOption(Opts[Opt.VERSIONS_CHROME])
     .addOption(Opts[Opt.VERSIONS_ANDROID])
+    .addOption(Opts[Opt.VERSIONS_APPIUM])
     .addOption(Opts[Opt.CHROME_LOGS])
     .addOption(Opts[Opt.ANDROID])
+    .addOption(Opts[Opt.IOS])
```
Comment (cnishina) [inline lib/cmds/start.ts]: Check to see if the user is on os.type() === 'Darwin' before adding the iOS options.

File: lib/cmds/update.ts (hunk 2 above)
Comment (sjelin) [inline lib/cmds/update.ts]: Done

**Pull request merged**